spec-0008/index.md

```diff
@@ -0,0 +1,228 @@
+# SPEC-8: Securing the Release Process
+
+---
+
+title: "SPEC 8 — Securing the Release Process"
+number: 8
+date: 2024-06-04
+author:
+
+- "Matthew Feickert <matthew.feickert@cern.ch>"
+- "Pamphile Roy <roy.pamphile@gmail.com>"
+- "Juanita Gomez <juanitagomezr2112@gmail.com>"
+- "Seth Larson <sethmichaellarson@gmail.com>"
+- "Lars Grüter <lagru@mailbox.org>"
+  discussion: https://discuss.scientific-python.org/t/spec-8-supply-chain-security
+  endorsed-by:
+
+---
+
+## Description
+
+<!--
+Briefly and clearly describe the recommendation.
+https://github.com/scientific-python/summit-2024/issues/9
+-->
+
+Open source libraries constitute a significant portion of the world's digital infrastructure. Securing the Open Source supply chain (OSSC) is therefore an increasing concern, with examples of sophisticated attacks against the ecosystem (e.g., the 2024 [`xz` utils backdoor](https://en.wikipedia.org/wiki/XZ_Utils_backdoor)) and [malware attacks on PyPI](https://blog.pypi.org/posts/2024-04-10-domain-abuse/) highlighting the need for supply chain security to be taken seriously.
+The Python Software Foundation (PSF) is also taking the importance of the OSSC seriously, as demonstrated by the [creation of the PSF Security Developer in Residence position in 2023](https://pyfound.blogspot.com/2023/06/announcing-our-new-security-developer.html).
+
+With the [Supply-chain Levels for Software Artifacts (SLSA) framework](https://slsa.dev/) and [OpenID Connect (OIDC)](https://openid.net/developers/how-connect-works/) standard being widely adopted, several high level developer tools, maintained by professional security teams, have been created with clear recommendations on how to use them.
+
+This SPEC outlines pragmatic recommendations for adopting these security tools, and recommendations on how to publish release artifacts securely.
+Securely *building* release artifacts will be covered in a later SPEC. This set of recommendations complements the recommendations from [SPEC 6 — Keys to the Castle](https://github.com/scientific-python/specs/blob/main/spec-0006/index.md).
+
+While this SPEC is written with GitHub in mind, the same recommendations apply to other services, such as GitLab.
+
+## Implementation
+
+With a focus on securing the release artifact distribution process, the following processes and standards should be adopted.
+
+### Document the release process
+
+The release process should be clearly and fully documented in the developer documentation and describe each step to make a release and the permissions required to do so.
+It is recommended that this is a dedicated page in the developer section of the documentation website, though providing instructions in a `RELEASING.md` in the top level of the repository is also a common approach.
+
+### Hardening workflow environment permissions
```

Comment:
@bsipocz Sorry, but just so I make sure I understand which are the recommendations you want bulleted? We already have things as subsections with `###` headers, but maybe you mean something else? Could you either point specifically to which lines or make a suggested change?

Reply:
The document has not been rendered just yet, so it's a bit difficult to judge how it will look like, but as I was reading this serially, and was commenting on it as such, I felt that in this `###` section there are 4 unrelated things listed, so would have found having `*` listed them useful.